roles/network_plugin/cilium/tasks/install.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Cilium | Ensure BFPFS mounted
```

Comment:
BPFFS according to line 6?

Reply:
Yes, seems like it. But I didn't write this part, just moved to a different file. Should we fix it in the same PR?

Reply:
If you could yes :)

Reply:
Done! :)